css/aioseop-jquery-ui.css

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
Font size and color styles are missing, which would be from `.ui-widget` and `.ui-widget-content`.

This is what was applied (on post-edit screen) prior to these PR changes.
![PR2655](https://user-images.githubusercontent.com/2794445/59542834-c44eb880-8ebc-11e9-988b-c65c6166a554.PNG)

Reply:
@EkoJR I took that out because it fixes a non-reported bug where some of the text in the metabox is bigger than intended. Good catch though. :-)